packages/mds-transaction-service/tests/transactions.spec.ts

```diff
@@ -258,6 +260,93 @@ describe('Transaction Service Tests', () => {
             return currTransaction
           }, undefined)
         })
+
+        it('Get Bulk Transactions with search string on receipt json', async () => {
+          const limit = 5
+          const provider_id = uuid()
+
+          const transactionsToPersist = [
+            ...transactionsGenerator(5, { provider_id, receipt_details: { color: 'Blue' } })
+          ] // Arbitrarily generate 5 events
+          await TransactionServiceClient.createTransactions(transactionsToPersist)
+
+          // Partial matching on string-value 'blu'
+          const { transactions: bluePages } = await TransactionServiceClient.getTransactions({
+            limit,
+            search_text: 'blu'
+          })
+          expect(bluePages.length).toEqual(limit) // all results are color:blue
+
+          // Search for a non-existent value
+          const { transactions: redPages } = await TransactionServiceClient.getTransactions({
+            limit,
+            search_text: 'red'
+          })
+          expect(redPages.length).toEqual(0) // no results are color:red
+
+          await TransactionServiceClient.createTransactions([
+            ...transactionsGenerator(5, {
+              provider_id,
+              receipt_details: { whitePages: 5, releaseDate: '1993/09/24', color: 'White' }
+            })
+          ])
+
+          // Search for a numeric value
+          const { transactions: whitePages } = await TransactionServiceClient.getTransactions({
+            limit,
+            search_text: '5'
+          })
+          expect(whitePages.length).toEqual(limit) // found all the white pages missing from the book!
+        })
+
+        it.only('Get Bulk Transactions with search on fee_type, amount', async () => {
+          const limit = 5
+          const provider_id = uuid()
+
+          const transactionsToPersist = [
+            ...transactionsGenerator(5, { provider_id, receipt_details: { color: 'Blue' }, fee_type: 'parking_fee' })
+          ] // Arbitrarily generate 5 events
+          await TransactionServiceClient.createTransactions(transactionsToPersist)
+
+          // Partial matching on string-value 'blu', fee_type is parking_fee
+          const { transactions: bluePages } = await TransactionServiceClient.getTransactions({
+            limit,
+            search_text: 'blu',
+            fee_type: 'parking_fee'
+          })
+          expect(bluePages.length).toEqual(limit) // all results are color:blue
+
+          // Search for a non-existent value
+          const { transactions: redPages } = await TransactionServiceClient.getTransactions({
```

Comment:
Can we change `redPages` to `baseFeePages`?

Reply:
fixy!